### DISCUSSION TITLE
Ошибка при установке.

### DISCUSSION THREAD

**Discussion by ostap7474** [vmakeev/huawei_mesh_router#12] [Q&A] [answered]
Ошибка при установке:  
Ошибка
Эта интеграция не поддерживает настройку через пользовательский интерфейс. Если Вы перешли по этой ссылке с веб-сайта Home Assistant, убедитесь, что у Вас установлена последняя версия Home Assistant. Установлен Home Assistant 2022.11.4

**Comment by vmakeev** [comment, marked as answer]
Странно. Давайте проверим корректность установки интеграции по шагам:

1. Скачать архив https://github.com/vmakeev/huawei_mesh_router/archive/refs/tags/v0.7.4.zip
2. Распаковать папку `custom_components` в папку с конфигурацией HomeAssistant (в итоге должен получиться такой путь: `<папка_с_вашим_конфигом_hass_где_лежит_configuration.yaml>/custom_components/huawei_mesh_router`
3. Перезапустить Home Assistant (без перезапуска новая интеграция может корректно не подтянуться)
4. Добавить новую интеграцию через `Настройки -> Устройства и службы -> Добавить интеграцию -> Huawei Mesh Router`  (или по ссылке типа `http://<адрес_вашего_ui_home_assistant>/_my_redirect/config_flow_start?domain=huawei_mesh_router`)

Только что проверил на `2022.11.4`, настройка через UI запустилась

**UPD**: или вы пробуете установить её через HACS?

**Comment by ostap7474** [comment]
Всё получилось, я был не внимателен и поместил папку huawei_mesh_router-0.7.4 в custom_components. Спасибо за помощь.
